sdk/bindings/swift/src/lib.rs

```diff
@@ -203,7 +203,7 @@ pub mod ffi {
         pub amount: f64,
         pub network: String,
         pub status: String,
-        pub explorer_url: String,
+        pub explorer_url: Vec<String>,
```

Comment:
I do not think we need to support multiple explorer urls, it is only the node urls that we need to support 💯 
```suggestion
        pub explorer_url: String,
```

Reply:
I saw that we were assigning `node urls` to the `explorer_url` and that's the chain of events that resulted in this change. I reverted this back and added some comments to fix this and use the `Network` type instead of some hardcoded values 😄